salt_resources/states/venv/pip_requirements.sls

```diff
@@ -1,7 +1,7 @@
-install_pip_requirements:
+install_venv_pip_requirements:
   cmd.run:
-    - require:
-      - sls: venv
     - name: ./bin/pip install -r /home/{{ grains['deescalated_user'] }}/{{ grains['project'] }}/{{ grains['pip_req_path'] }}
     - cwd: /home/{{ grains['deescalated_user'] }}/venv/
```

Comment:
I'm suspicious this line is unnecessarily. We supply the bin path and req file path in the `- name`. I'll add a ticket to investigate it.